### DISCUSSION TITLE
Buttons in a message?

### DISCUSSION THREAD

**Discussion by GIFUS** [Rapptz/discord.py#7267] [Q&A]
Can anyone give an example of adding buttons to embed?
I will be grateful.

**Comment by daggy1234** [comment]
Just pass in embed as a kwarg when sending the view with the buttons. For example 

```py
await ctx.send(embed=embed_here, view=view_here)
```

and it should work!

[Docs](https://discordpy.readthedocs.io/en/stable/api.html?highlight=send#discord.abc.Messageable.send)

**Reply by GIFUS** [reply]
I tried to make buttons according to some examples. And I get the error `AttributeError: module 'discord' has no attribute 'ui'`
Do you know what is the problem?

**Reply by lgaan** [reply]
You need to install the master version of the library via `pip/pip3 install -U git+https://github.com/Rapptz/discord.py`

**Reply by daggy1234** [reply]
You need to use beta dpy 2.0. This is on master only at the moment. I would suggest joining the discord [Discord](https://discord.gg/dpy) for more information.
